home/home_2.2/test_script.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
```

Comment:
the same comment as for class work - use extra dataset files to have input data and expected result to simplify the logic of test and make it more flexible